src/sentry/migrations/0188_backfill_date_added_on_release_project.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 1.11.29 on 2021-04-30 13:08
+
+from django.db import migrations, models
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_date_added(apps, schema_editor):
+    """
+    Fill the new ReleaseProject.date_added column with values from the Release.date_added
+    """
+    ReleaseProject = apps.get_model("sentry", "ReleaseProject")
+    Release = apps.get_model("sentry", "Release")
+    all_releases = Release.objects.all()
+    for release in RangeQuerySetWrapperWithProgressBar(all_releases):
+        release_project = ReleaseProject.objects.get(release=release)
+        release_project.date_added = release.date_added
+        release_project.save()
```

Comment:
Shouldn't this backfill happen *after* you release code that starts to write to this column? Right now it looks like the following would happen:

1. The column is added.
2. The backfill migration is run.
3. A non-zero amount of time passes as workers are deployed. This step can take 10-20min in practice.
4. The web workers restart and start writing new releaseprojects with `date_added`.

Between steps 3 and 4 there will be releaseproject records that do not have `date_added` set. This would require another backfill to resolve.

Another approach you could take is

1. Release a migration to add the `date_added` column.
2. Update endpoints to start populating this data so all new rows are correct.
3. Release the backfill to update all historical records.